examples/opengl_boxes.rs

```diff
@@ -26,6 +26,8 @@ use limn::input::{EscKeyCloseHandler, DebugSettingsHandler};
 use limn::widgets::slider::{SliderBuilder, SliderEvent};
 use limn::widgets::glcanvas::{GLCanvasBuilder, GLCanvasState};
 
+pub const FLOATING_PT_ERROR: f32 = 0.0001;
```

Comment:
Normally should use std::f32::EPSILON for this use case, however in this case strict float equality is actually what is intended so neither are needed